src/scripts/run_floodscan_pipeline.py

```diff
@@ -36,6 +36,13 @@ def parse_arguments(base_parser):
         choices=[5],
         default=5,
     )
+    parser.add_argument(
+        "--baseline-update",
+        "-b",
+        help="Add baseline for 10 years prior to some specific year",
```

Comment:
Would clarify a bit further here on how this parameter needs to be specified and what it means. So you input a year (eg. 2024), and the baseline file for 2014-2024 is generated?

Reply:
Could you also add info about the new parameter to this doc: https://github.com/OCHA-DAP/ds-raster-pipelines/blob/main/docs/usage.md